docs/guide/rasa/custom_actions.mdx

```diff
@@ -48,92 +48,9 @@ In the `run` method you can access the conversation state and utter responses. T
 
 ## Using Custom actions
 
-Whether you're a Botfront Open Source user or a Botfront Cloud customer creating custom actions is super easy.
+Whith Botfront Open Source, creating custom actions is super easy.
 As explained below, you can spin up an action server in no time with the CLI and your code will be hot reloading so you can see the impact of your changes live.
 
-### In Botfront Cloud
-
-#### Development
-
-To develop your actions, the easiest is to run an action server locally and tunnel Botfront.
-
-To start an action server locally:
-
-1. Install the Botfront CLI:
-```
-npm install -g botfront
-```
-
-2. Create an action projects:
-```
-botfront init --cloud
-```
-
-3. `cd` the project folder and launch the action server
-```
-botfront up
-```
-
-4. Watch for file changes
-```
-botfront watch
-```
-
-Any change in the `actions` folder will rebuild and restart the actions server.
-
-
-When you save an action, you can verify that it's loaded in the logs by looking for this line:
-```
-INFO:rasa_sdk.executor:Registered function for 'action_fetch_status'.
-```
-
-5. In a different terminal (same location) you can follow the logs:
-```
-botfront logs
-```
-
-You can create your actions in the `actions` folder.
-You can add actions to the default `my_actions.py` file or add new files.
-You can use the `Dockerfile` to install Python dependencies.
-
-The last thing you need to do is to create a tunnel to get your Rasa instance on Botfront Cloud call your local actions server. You can use **Ngrok** for that.
-
-[Sign up](https://dashboard.ngrok.com/signup) to a free Ngrok plan.
-
-[Download and setup Ngrok](https://ngrok.com/download) on your machine.
-
-The installation is not global, which means you'll need to invoke `ngrok` with the full path. To avoid that create a symlink (MacOS and Linux)
-
-```bash
-# cd into your local bin directory
-cd /usr/local/bin
-# create symlink to the installation path
-ln -s /Applications/ngrok ngrok
-```
-
-To start **Ngrok**, run `ngrok http 5055`. `5055` is the port used by the actions server on your local machine.
-
-You will see the following in your console:
-![](../../images/actions_ngrok.png)
-
-Copy the forward URL and past it in Settings -> Endpoints. **Don't forget the `/webhook` route**
-![](../../images/actions_settings.png)
-
-<Important type="tip" title="Importing modules in your actions">
-
-Your imports must include `actions`. If you want to import a module in the `actions` folder:
-
-```python
-import actions.my_module
-```
-
-</Important>
-
-#### Production
-
-Contact us if you want us to host your production actions deployment.
-
-### Botfront Open Source
```

Comment:
I removed that quickly as it was mentioning Botfront cloud But I'm wondering If I should bring it back and reword it a bit for os ?